docs/content/docs/tutorials/eventbridge-example.md

```diff
@@ -0,0 +1,308 @@
+export TARGET_QUEUE=custom-eventbus-ack-rule-sqs-target
+```
+
+Create the target queue.
+
+```bash
+cat <<EOF > target-queue.json
+{
+    "QueueName": "${TARGET_QUEUE}",
+    "Attributes": {
+        "Policy": "{\"Statement\":[{\"Sid\":\"EventBridgeToSqs\",\"Effect\":\"Allow\",\"Principal\":{\"Service\":\"events.amazonaws.com\"},\"Action\":[\"sqs:SendMessage\"],\"Resource\":\"arn:aws:sqs:${AWS_REGION}:${AWS_ACCOUNT_ID}:${TARGET_QUEUE}\",\"Condition\":{\"ArnEquals\":{\"aws:SourceArn\":\"arn:aws:events:${AWS_REGION}:${AWS_ACCOUNT_ID}:rule/${EVENTBUS_NAME}/${RULE_NAME}\"}}}]}"
+    }
+}
+EOF
+
+aws sqs create-queue --cli-input-json file://target-queue.json
+```
+
+The output of above commands looks like
+
+```bash
+{
+    "QueueUrl": "https://sqs.us-east-1.amazonaws.com/1234567890/custom-eventbus-ack-rule-sqs-target"
+}
+```
+
+### Create a Custom Event Bus
+
+Execute the following command to create the example namespace and a custom event bus.
+
+```bash
+kubectl create ns ${EVENTBRIDGE_NAMESPACE}
+
+cat <<EOF > bus.yaml
+apiVersion: eventbridge.services.k8s.aws/v1alpha1
+kind: EventBus
+metadata:
+  name: ${EVENTBUS_NAME}
+spec:
+  name: ${EVENTBUS_NAME}
+EOF
+
+kubectl -n ${EVENTBRIDGE_NAMESPACE} create -f bus.yaml
+```
+
+The output of above commands looks like
+
+```bash
+namespace/eventbridge-example created
+eventbus.eventbridge.services.k8s.aws/custom-eventbus-ack created
+```
+
+Verify the event bus resource is synchronized.
+
+```bash
+kubectl -n ${EVENTBRIDGE_NAMESPACE} get eventbus ${EVENTBUS_NAME}
+```
+
+The output of above commands looks like
+
+```bash
+NAME                  SYNCED   AGE
+custom-eventbus-ack   True     64s
+```
+
+### Create a Rule with an SQS Target
+
+Execute the following command to retrieve the ARN for the SQS target created above needed for the Kubernetes manifest.
+
+```bash
+export TARGET_QUEUE_ARN=$(aws --output json sqs get-queue-attributes --queue-url "https://sqs.${AWS_REGION}.amazonaws.com/${AWS_ACCOUNT_ID}/${TARGET_QUEUE}" --attribute-names QueueArn | jq -r '.Attributes.QueueArn')
+```
+
+Execute the following command to create a Kubernetes manifest for a rule, forwarding events matching the specified rule
+filter criteria to the target queue. The EventBridge filter pattern will match any event received on the custom event
+bus with a `detail-type` of `event.from.ack.v0`. Alternatively, the filter pattern can be omitted to forward all events
+from the custom event bus.
+
+```bash
+cat <<EOF > rule.yaml
+apiVersion: eventbridge.services.k8s.aws/v1alpha1
+kind: Rule
+metadata:
+  name: $RULE_NAME
+spec:
+  name: $RULE_NAME
+  description: "ACK EventBridge Filter Rule to SQS using event bus reference"
+  eventBusRef:
+    from:
+      name: $EVENTBUS_NAME
+  eventPattern: |
+    {
+      "detail-type":["event.from.ack.v0"]
+    }
+  targets:
+    - arn: $TARGET_QUEUE_ARN
+      id: sqs-rule-target
+      retryPolicy:
+        maximumRetryAttempts: 0 # no retries
```

Comment:
eventually once we support dynamic reference typing, we should be able to reference an SQS Queue by Kubernetes resource name here, yes? :)

Reply:
EXACTLY! Can't wait! `dynamic.Client` #FTW :D